packages/database/drizzle/0058_medical_plazm.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS "draft_projects" (
+	"draft_id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"project_name" varchar(255),
+	"token_symbol" varchar(15),
+	"token" varchar(15),
+	"operator_name" varchar(255),
+	"project_image_url" varchar(1023),
+	"website_url" varchar(511),
+	"short_description" text,
+	"long_description" text,
+	"minimum_raise" numeric(20, 2),
+	"monthly_spending" numeric(20, 2),
+	"team_wallets" jsonb,
+	"multisig_threshold" integer,
+	"premine_amount" numeric(20, 2),
+	"performance_package_unlock_destination" varchar(44),
+	"unlock_months" integer,
+	"legal_terms" text,
+	"metalex_tx_hash" varchar(127),
+	"terms_of_service_url" text,
```

Comment:
let's add and can be nullable except the name of the founder:

- `ideal_start_date` date nullable
- `token_address` varchar(XXX) nullable
- `token_seed` varchar(xx) nullable
- `project_owners_name` text (can name it something else, like founders_name)
- `ip_details` text  (this is where we'll put the IP they expect to transfer)
- `existing_investors` jsonb nullable (this will be used in future versions)
- `existing_investors_allocation` numeric (this is % allocated for the vesting carveout)
- `socials` text... or jsonb